sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/datalake_options.hpp

```diff
@@ -218,9 +218,9 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
     Azure::Core::Context Context;
 
     /**
-     * @brief Specify the transactional md5 for the body, to be validated by the service.
+     * @brief Specify the transactional hash for the body, to be validated by the service.
      */
-    Azure::Core::Nullable<ContentHash> ContentMd5;
+    Azure::Core::Nullable<Storage::ContentHash> TransactionalContentHash;
```

Comment:
Do we need `Storage::` here?

Reply:
ditto for the others

Reply:
Some parameter names are `ContentHash`, this avoids compile failure.

Reply:
I get it.